test/cli/kit/config_test.rb

```diff
@@ -90,7 +90,8 @@ def test_config_set
         assert_equal("[section]\nsome-key = ~/.test\nsome-other-key = ~/.test", File.read(@file))
 
         assert_equal('~/.test', @config.get('section', 'some-key'))
-        assert_equal("#{ENV["HOME"]}/.test", @config.get_path('section', 'some-key'))
+        assert_equal(File.expand_path("#{ENV["HOME"]}/.test"),
+          File.expand_path(@config.get_path('section', 'some-key')))
```

Comment:
The point of this test was to check that `get_path` expands the path.
```suggestion
        assert_equal(File.expand_path('.test', ENV['HOME']), @config.get_path('section', 'some-key'))
```